Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM nfcore/base:1.9
+FROM nfcore/base:1.12.1
 LABEL authors="Payam Emami" \
-      description="Docker image containing all requirements for nf-core/metaboigniter pipeline"
+      description="Docker image containing all software requirements for the nf-core/metaboigniter pipeline"
 
 # Install the conda environment
-COPY environment.yml /
-RUN conda env create -f /environment.yml && conda clean -a
+#COPY environment.yml /
+#RUN conda env create --quiet -f /environment.yml && conda clean -a
```

Comment:
Ah because you have no conda environment. In that case you should not have a main `Dockerfile` either if the image this is generating is not used in the pipeline.